oauth2client/crypt.py

```diff
@@ -248,3 +237,25 @@ def verify_signed_jwt_with_certs(jwt, certs, audience=None):
     _check_audience(payload_dict, audience)
 
     return payload_dict
+
+
+def configure_module():
+    """Configure module-wide variables.
+
+    Not all testing frameworks load modules directly, so we need a way to
+    define these on set up.
+    """
+    global Signer
+    global Verifier
```

Comment:
They were global before, just being defined at the module level.  Pytest doesn't load module/global variables, which is why it was failing (unless someone knows of a way to do that).

I'll see if I can come up with a cleaner/better way to do that.

I came across [this answer](http://stackoverflow.com/a/7668273/398514) on StackOverflow regarding `__getattr__` at the module level, which referenced a hack that [Guido van Rossum suggested](https://mail.python.org/pipermail/python-ideas/2012-May/014969.html).